spec/classes/virtualbox_spec.rb

```diff
@@ -34,22 +34,16 @@
       # Debian specific stuff
       #
       if facts[:osfamily] == 'Debian'
-        context 'with $::puppetversion < 3.0.0' do
-          let(:facts) { facts.merge(puppetversion: '2.7.26') }
 
-          it { is_expected.not_to contain_class('apt') }
-          it { is_expected.not_to contain_apt__source('virtualbox') }
-        end
-
-        unless Puppet::Util::Package.versioncmp(facts[:puppetversion], '3.0.0') == -1
+        unless Puppet::Util::Package.versioncmp(facts[:puppetversion], '4.0.0') == -1
```

Comment:
we can get rid of this line. Our testmatrix only uses puppet 4.10 and puppet 5.  It is defined in https://github.com/voxpupuli/modulesync_config/blob/master/config_defaults.yml

Reply:
done